### DISCUSSION TITLE
Winmerge says left only but manual review shows identifical files in right

### DISCUSSION THREAD

**Discussion by hrrrps** [WinMerge/winmerge#2179] [Q&A] [answered]
I am comparing two folders with extensive subfolders (with subfolders etc.), so the number of files is quite extensive. The folders and files are clearly mostly identical (random sampling of folders and files) yet Winmerge is showing left only. This has never happened to me before. Could there be a problem with my settings?

**Comment by sdottaka** [comment]
Do the folder or file names contain diacritical marks? If so, the file or folder name may have a name that looks the same but has a different internal representation.
Related Discussion: https://github.com/WinMerge/winmerge/discussions/2121
If not, could you please attach a screenshot or a compressed file of folders that can be reproduced?

**Reply by hrrrps** [reply, marked as answer]
Human error, sorry.
